From b3ca1d02bb0148ca787ac4aead164d7c8ce2c4d8 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Wed, 12 Jan 2022 00:24:20 +0100
Subject: [PATCH] meson: Fix unknown kw argument in gnome.generate_gir

This argument has been removed in Meson 0.61.0:

    gck/meson.build:130:2: ERROR: gnome.generate_gir got unknown keyword arguments "packages"

https://github.com/mesonbuild/meson/commit/f8fc5cb860465718fe7c79a1bf1fe00659f138de:

> The packages argument to gnome.generate_gir was allowed, but never did anything, so stop passing it.

Fixes #89

[aperez@igalia.com: backport from upstream]
Signed-off-by: Adrian Perez de Castro <aperez@igalia.com>
[yann.morin.1998@free.fr: make it an actual backport]
Signed-off-by: Yann E. MORIN <yann.morin.1998@free.fr>
---
 gck/meson.build | 1 -
 gcr/meson.build | 1 -
 ui/meson.build  | 1 -
 3 files changed, 3 deletions(-)

diff --git a/gck/meson.build b/gck/meson.build
index 756b486..a21a1e9 100644
--- a/gck/meson.build
+++ b/gck/meson.build
@@ -131,7 +131,6 @@ if get_option('introspection')
     sources: gck_gir_sources,
     namespace: 'Gck',
     nsversion: '@0@'.format(gck_major_version),
-    packages: gck_deps,
     export_packages: 'gck-@0@'.format(gck_major_version),
     includes: [ 'GObject-2.0', 'Gio-2.0' ],
     header: 'gck/gck.h',
diff --git a/gcr/meson.build b/gcr/meson.build
index 2233a44..c83641b 100644
--- a/gcr/meson.build
+++ b/gcr/meson.build
@@ -190,7 +190,6 @@ if get_option('introspection')
     sources: [ gcr_base_public_sources, gcr_base_headers ],
     namespace: 'Gcr',
     nsversion: '@0@'.format(gcr_major_version),
-    packages: gcr_base_deps,
     export_packages: 'gcr-base-@0@'.format(gcr_major_version),
     includes: [
       'GObject-2.0',
diff --git a/ui/meson.build b/ui/meson.build
index e656ea2..32ee057 100644
--- a/ui/meson.build
+++ b/ui/meson.build
@@ -152,7 +152,6 @@ if get_option('introspection')
     export_packages: 'gcr-ui-@0@'.format(gcr_major_version),
     identifier_prefix: 'Gcr',
     symbol_prefix: 'gcr',
-    packages: gcr_ui_deps,
     includes: [
       'GObject-2.0',
       'Gio-2.0',
-- 
2.25.1

